.github/workflows/release-notes-labels.yml

```diff
@@ -0,0 +1,22 @@
+name: Enforce labels for release notes
+on:
+  pull_request:
+    types: [opened, labeled, unlabeled, synchronize]
+jobs:
+  label:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+    steps:
+      - uses: mheap/github-action-required-labels@d25134c992b943fb6ad00c25ea00eb5988c0a9dd
+        with:
+          mode: minimum
+          count: 1
+          labels: |
+            ignore-for-release
+            new-template
+            improvement
+            bug-fix
+          add_comment: true
+          message: "This PR is being prevented from merging because you have not added any of the following labels: [ignore-for-release, new-template, improvement, bug-fix]. You'll need to add one before this PR can be merged so that these can be used for release notes."
```

Comment:
let us make sure we update https://github.com/GoogleCloudPlatform/DataflowTemplates/tree/main/contributor-docs to call this out.

Reply:
Done, thanks!